src/components/Header.jsx

```diff
@@ -1,11 +1,26 @@
-import Topics from './topics/Topics';
-import NavBar from './NavBar';
+import { Link } from 'react-router-dom';
+import Topics from './Topics';
 
 const Header = () => {
 	return (
 		<>
 			<h1>NC News</h1>
-			<NavBar />
+			<nav className="navbar navbar-expand-lg bg-body-tertiary">
```

Comment:
You have a Header component that is essentially a nav. Is there any reason you abandoned the component you had before?